### PR TITLE
Blood Bag Highlight - Quick Items.

### PR DESCRIPTION
**Pull request by Sashank999**

### CODE REVIEW

File: extension/scripts/content/factions/ttFactions.js [hunk 1]
```diff
@@ -76,7 +76,7 @@ const isOwnFaction = getSearchParameters().get("step") === "your";
 
 		triggerCustomListener(EVENT_CHANNELS.FACTION_ARMORY_TAB, { section: getCurrentSection() });
 		new MutationObserver((mutations) => {
-			if (mutations.length < 2) return;
+			if (mutations.addedNodes?.length < 2) return;
```
Comment (DeKleineKobini) [inline extension/scripts/content/factions/ttFactions.js:79]: Why this? It worked perfectly before?

File: extension/scripts/features/highlight-blood-bags/ttHighlightBloodBags.js [hunk 2]
```diff
@@ -87,10 +88,10 @@
 	}
 
 	async function removeHighlights() {
-		for (const highlight of document.findAll(".good-blood, .bad-blood")) {
+		for (const highlight of document.findAll(".items-wrap .good-blood, .items-wrap .bad-blood, #armoury-medical .good-blood, #armoury-medical .bad-blood")) {
```
Comment (DeKleineKobini) [inline extension/scripts/features/highlight-blood-bags/ttHighlightBloodBags.js:91]: Why make it more specific? Those classes are only used for this feature anyway.

**Review by DeKleineKobini — CHANGES_REQUESTED** [review]

**Comment by DeKleineKobini** [comment]
And I'm not sure about the feature in itself.

**Comment by DeKleineKobini** [comment]
I'll let Mephiles decide whether we should include this change.

File: extension/scripts/content/factions/ttFactions.js (hunk 1 above)
Comment (Sashank999) [reply extension/scripts/content/factions/ttFactions.js:79]: IDK. It works like this now.

File: extension/scripts/features/highlight-blood-bags/ttHighlightBloodBags.js (hunk 2 above)
Comment (Sashank999) [reply extension/scripts/features/highlight-blood-bags/ttHighlightBloodBags.js:91]: It removes the classes for Quick Items items too.

**Comment by DeKleineKobini** [comment]
We've decided to not include this for now. Should there come an actual demand for this, we'll figure out on how to implement it then.

**Pull request closed without merging**